schemas/network.yml

```diff
@@ -10,16 +10,6 @@
       description: >
         Network protocol name.
       example: http
-    - name: direction
```

Comment:
I think this field we still need?

Reply:
@ruflin src and dest are explicit in the direction. This is redundant imo.

Reply:
It's a convention field to filter on. We for example use it here in auditbeat: https://github.com/elastic/beats/blob/master/auditbeat/module/auditd/_meta/fields.yml#L162

@andrewkroh can perhaps share some more background here?